mesh/mesh.hpp

```diff
@@ -583,7 +583,10 @@ class Mesh
    /** Each vertex @a i will be mapped to vertex @a v2v[i], such that all
        vertices that are coincident under the periodic mapping get mapped to
        the same index. The mapping @a v2v can be generated from translation
-       vectors using Mesh::CreatePeriodicVertexMapping. */
+       vectors using Mesh::CreatePeriodicVertexMapping.
+       @note MFEM requires that each edge of the resulting mesh be uniquely
+       identifiable by a pair of unique vertiecs. As a consequence, periodic
```

Comment:
Nicely stated but there is a small typo "vertiecs".

Reply:
Fixed, thanks.